single-audit/resolve_findings/templates/resolve_findings/finding-resolution-page.html

```diff
@@ -120,60 +120,24 @@ <h2 class="usa-accordion__heading">
           {% endwith %}
         </div>
 
-        <!-- @alexsoble, what would you think of this subsection (comments)
```

Comment:
@fureigh Removed this note because this PR implements a separate subsection div, below.